core/shared/src/main/scala/zio/prelude/ZValidation.scala

```diff
@@ -193,8 +193,11 @@ sealed trait ZValidation[+W, +E, +A] { self =>
   /**
    * Converts this `ZValidation` into a `ZIO` effect, discarding the log.
    */
-  final def toZIO: IO[NonEmptyChunk[E], A] =
-    ZIO.fromEither(self.toEither)
+  final def toZIO: IO[E, A] =
+    self.fold(
+      nec => ZIO.halt(nec.foldLeft[zio.Cause[E]](zio.Cause.empty)((c, e) => zio.Cause.Both(c, zio.Cause.fail(e)))),
```

Comment:
Can we use `reduceMapLeft` here so we don't have to handle the empty case?

Reply:
Nice!